.github/workflows/tests-docker.yml

```diff
@@ -26,35 +26,29 @@ jobs:
         cp tests/test.env dev.env
         cat dev.env
         python3 wis2box-ctl.py config
-    - name: install loki plugin for docker
-      run: | 
-        docker plugin install grafana/loki-docker-driver:latest --alias loki --grant-all-permissions
-        docker plugin ls
-    - name: build wis2box images and start containers ⚙️
+    - name: build wis2box
       run: |
         python3 wis2box-ctl.py build
+    - name: start containers ⚙️
+      run: |
         python3 wis2box-ctl.py start
-        sleep 5
-        python3 wis2box-ctl.py status
+        sleep 15
+        python3 wis2box-ctl.py status -a
     - name: test wis2box data ingest/process/publish workflow ⚙️
       run: |
         docker exec wis2box sh -c " \
           wis2box environment create \
           && wis2box environment show \
-          && wis2box data setup --topic-hierarchy data.core.observations-surface-land.mw.FWCL.landFixed \
-          && wis2box data info --topic-hierarchy data.core.observations-surface-land.mw.FWCL.landFixed \
```

Comment:
Let's keep `wis2box data info` in the CI?  Might be valuable for debugging sane paths in the future.